content/en/_index.html

```diff
@@ -3,50 +3,50 @@
 linkTitle = "Open Modeling Foundation"
 +++
 
-{{< blocks/cover image_anchor="bottom" color="primary" height="full" >}}
-<h1>The Open Modeling Foundation</h1>
-<h3 class="mb-3 pt-3 font-weight-light">Enabling next generation modeling of human and natural systems</h3>
-
-<div class="mx-auto py-5">
-    <a class="btn btn-lg btn-outline-light mr-3 mb-4" href="{{< relref "/about" >}}">
-    About <i class="fas fa-info-circle ml-2"></i>
-    </a>
-    <a class="btn btn-lg btn-outline-light mr-3 mb-4" href="{{< relref "/governance" >}}">
-    Organization & Governance <i class="fas fa-gavel ml-2"></i>
-    </a>
-    <a class="btn btn-lg btn-outline-light mb-4" href="{{< relref "/standards" >}}">
-    Standards <i class="fas fa-lightbulb ml-2"></i>
-    </a>
-    <div class="mx-auto mt-5">
-        {{< blocks/link-down color="info" >}}
+{{< blocks/section color="primary" >}}
+  <div class="row justify-content-center mt-5 mb-3">
+    <div class="col-md-10 col-lg-8 d-flex flex-column align-items-center">
+      <img src="/images/logo.svg" alt="OMF Logo" class="w-50 img-fluid mb-5">
+      <h3 class="font-weight-light text-center">
+        The <b class="text-secondary">Open Modeling Foundation</b> is an alliance of modeling organizations that coordinates
+        and administers a common, community developed body of <b>standards</b> and <b>best practices</b> among
+        diverse communities of modeling scientists.
+      </h3>
     </div>
-</div>
-
-
-{{< /blocks/cover >}}
-
-
-{{% blocks/lead color="white" %}}
-<div class="mx-auto">
-    <h3 class="mt-5 font-weight-light text-left">The Open Modeling Foundation is an alliance of modeling organizations that coordinates and administers a common, community developed body of standards and best practices among diverse communities of modeling scientists.
-    </h3>
-</div>
-{{% /blocks/lead %}}
+  </div>
+{{< /blocks/section >}}
 
-{{< blocks/section color="secondary" >}}
+{{< blocks/section color="dark" >}}
+  <div class="col-12 text-center mb-5"><h2><b>Standards</b></h2></div>
+  {{% blocks/standard icon="fas fa-search" title="Accessibility" url="/standards/accessibility" %}}
+  Enabling discovery and access for model code, documentation, and metadata.
```

Comment:
Should we make these start with the verb instead of present participle? (I had to look that up)

So "Enable discovery and access ..." etc., for the others

Or we could change the "Facilitate reuse ..." to the present participle.

This is the most I've used "present participle" in the last 30 years probably.

Reply:
Ha, I think I prefer the base verb here. These blurbs, as usual, are free (and often encouraged) to be edited or rewritten by anyone better at this sort of thing